packages/documentation/docs/for-developers/json-config-schema.md

```diff
@@ -45,6 +51,28 @@ This is primarily for `json-schema-to-typescript`.
 
 Names of the enum values as generated for the typescript enum, which we display in the UI instead of the raw values
 
+### `ui:sofie-enum` & `ui:sofie-enum:filter`
+
+Note: Only valid for blueprint configuration.
+
+Sometimes it can be useful to reference other values. This property can be used on string fields, to let sofie generate a dropdown populated with values valid in the current context.
+
+#### `mappings`
+
+Valid for both show-style and studio blueprint configuration
+
+This will provide a dropdown of all mappings in the studio, or studios where the show-style can be used.
+
+Setting `ui:sofie-enum:filter` to an array will filter the dropdown by the specified DeviceType.
+
+#### `source-layers`
+
+Valid for only show-style blueprint configuration.
+
+This will provide a dropdown of all source-layers in the show-style.
+
+Setting `ui:sofie-enum:filter` to an array will filter the dropdown by the specified SourceLayerType.
```

Comment:
Specified how? I think we should be explicit here, that the `ui:sofie-enum:filter` takes an number array.